src/glossary.rst

```diff
@@ -9,21 +9,6 @@ Glossary
       A Cylc suite is a directory containing a :cylc:conf:`flow.cylc` file
       which contains :term:`graphing<graph>` representing a workflow.
 
-   suite directory
```

Comment:
Are we calling this the "suite directory" or the "source directory"? I had been working with the latter name, because:
1. We're trying to avoid the term suite, at least in terms of pure Cylc Stuff.
2. Source doesn't imply any connection to Rose, which is optional.

Reply:
I had removed `suite directory` after a discussion with Oliver. Is this ok @oliver-sanders?

Reply:
Sorry, me be daft, this is a removal - sorry.